crates/interpreter/src/instructions/host.rs

```diff
@@ -321,6 +327,24 @@ pub fn create<const IS_CREATE2: bool, SPEC: Spec>(
     }
 }
 
+pub fn create<const IS_CREATE2: bool, SPEC: Spec>(
+    interpreter: &mut Interpreter,
+    host: &mut dyn Host,
+) {
+    let mut create_input: Option<Box<CreateInputs>> = None;
+    prepare_create_inputs::<IS_CREATE2, SPEC>(interpreter, &mut create_input);
+
+    if create_input.is_none() {
+        return;
+    }
+
+    let mut create_input = create_input.unwrap();
+
+    let ret = host.create(&mut create_input);
+
+    handle_create_result(interpreter, ret);
```

Comment:
This is not needed as after `host.create` we are not in recursive call so it will not increment stack that much on recursion.